certbot/client.py

```diff
@@ -53,9 +54,15 @@ def determine_user_agent(config):
     """
 
     if config.user_agent is None:
-        ua = "CertbotACMEClient/{0} ({1}) Authenticator/{2} Installer/{3}"
-        ua = ua.format(certbot.__version__, util.get_os_info_ua(),
-                       config.authenticator, config.installer)
+        if cli.cli_command in ["certbot", "letsencrypt", "certbot-auto", "letsencrypt-auto"]:
+            auto = cli.cli_command
+        else:
+            auto = "other"
```

Comment:
My thinking was just that if the user moves our command to some other weird name like `certbot-sucks` or whatever, maybe that's private? But I guess it's corner case-y, and documented in the dummy UA, and the code is simpler without this logic.